### DISCUSSION TITLE
👋 Welcome!

### DISCUSSION THREAD

**Discussion by KBLiveSolutions** [KBLiveSolutions/open.control#10] [General]
We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GTraux** [comment]
Hey! My name is Gui. I'm an artist from Brazil based in Germany the last 5 years. I'm a music producer and live performer. You can find my work as Claxy in Spotify.

Pierre's SONG:LOOPER:MODE was a godsend for my duo live set tour of 2020! Our live setup this year changed again, and the new batch of Pocket open.control should be exactly what we need. Received last week and loving the clever programming and possibilities ahead.

All the best!
